Core/src/OffConverter/OffModel.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright (c) 2018 Technical University of Munich
```

Comment:
```suggestion
Copyright (c) 2021 Technical University of Munich
```

Similarly in other new files.

Reply:
Resolved: 2c1e522